source/configuration_manual/replication.rst

```diff
@@ -0,0 +1,484 @@
+   user was last modified. This doesn't mean that the sync succeeded
+   necessarily.
+
+-  full sync: How long time ago the last "full sync" attempt was
+   performed. This should happen once per
+   :ref:`setting-replication_full_sync_interval`.
+   This doesn't mean that the sync succeeded necessarily.
+
+-  success sync: Time when the last successful sync was performed. If
+   the last sync succeeded, this is the same as the "fast sync" or the
+   "full sync" timestap.
+
+-  failed: "y" if the last sync failed, "-" if not.
+
+The current dsync replication status can be looked up with
+``doveadm replicator dsync-status``. This shows the dsync replicator
+status for each potential dsync connection, as configured by
+:ref:`setting-replication_max_conns`. An example output is:
+
+::
+
+   username                   type   status
+   test100                    full   Waiting for dsync to finish
+   test1                      normal Waiting for handshake
+                              -      Not connected
+                              -      Not connected
+
+Here there are 4 lines, meaning ``replication_max_conns=4``. Only two of
+the dsync-connections are being used currently.
+
+The fields mean:
+
+-  username: User currently being replicated.
+
+-  type: incremental, normal or full. Most of the replications are
+   "incremental", while full syncs are "full". A "normal" sync is done
+   when incremental syncing state isn't available currently. The
+   "incremental" matches doveadm sync's -s parameter, "full" is -f
+   parameter and "normal" is the default.
+
+-  status: Human-readable status of the connection. These are the
+   current values:
+
+   -  Not connected
+
+   -  Failed to connect to '%s' - last attempt %ld secs ago
+
+   -  Idle
+
+   -  Waiting for handshake
+
+   -  Waiting for dsync to finish
+
+Failed replication attempts are always automatically retried, so any
+temporary problems should get fixed automatically. In case of bugs it
+may be necessary to fix something manually. These should be visible in
+the error logs. So if a user is marked as failed, try to find any errors
+logged for the user and see if the same error keeps repeating in the
+logs. If you want to debug the dsync, you can manually trigger it with:
+``doveadm -D sync -u user@domain -d -N -l 30 -U`` (the parameters after
+"sync" should be the same as in :ref:`setting-replication_dsync_parameters`
+setting).
+
+Notes
+-----
+
+Random things to remember:
+
+-  The replicas can't share the same quota database, since both will
+   always update it
+
+-  With mdbox format "doveadm purge" won't be replicated
+
+-  "doveadm force-resync", "doveadm quota recalc" and other similar
+   fixing commands don't get replicated
+
+-  The servers must have different hostnames or the locking doesn't work
+   and can cause replication problems.
+
+   -  v2.2.6+: If you're having trouble, verify that
+      ``dovecot --hostdomain`` returns different values for the servers.
```

Comment:
Remove v2.2.6+ (maybe add versionadded, although seems unnecessary complexity to me)